test/netsh.test.js

```diff
@@ -0,0 +1,94 @@
+const exampleIPAddressResult1 =
+`Wireless LAN adapter Wi-Fi:
+
+   Connection-specific DNS Suffix  . :
+   Description . . . . . . . . . . . : 1x1 11b/g/n Wireless LAN PCI Express Half Mini Card Adapter
+   Physical Address. . . . . . . . . : E1-23-E4-CE-C5-F6
+   DHCP Enabled. . . . . . . . . . . : Yes
+   Autoconfiguration Enabled . . . . : Yes
+   IPv6 Address. . . . . . . . . . . : 2001:db8:21da:7:713e:a426:d167:37ab(Preferred)
+   Temporary IPv6 Address. . . . . . : 2001:db8:21da:7:5099:ba54:9881:2e54
+   Link-local IPv6 Address . . . . . : fe80::713e:a426:d167:37ab%6
+   IPv4 Address. . . . . . . . . . . : 157.60.14.11(Preferred)
+   Subnet Mask . . . . . . . . . . . : 255.255.255.0
+   Default Gateway . . . . . . . . . :
+   DHCPv6 IAID . . . . . . . . . . . : 232785638
+   DHCPv6 Client DUID. . . . . . . . : 00-01-00-01-1E-99-E0-56-B8-88-E3-EA-F5-7A
+   DNS Servers . . . . . . . . . . . : fec0:0:0:ffff::1%1
+                                       fec0:0:0:ffff::2%1
+                                       fec0:0:0:ffff::3%1
+   NetBIOS over Tcpip. . . . . . . . : Enabled`;
+const exampleIPAddressResult2 =
+`Wireless LAN adapter Wi-Fi:
+
+   Connection-specific DNS Suffix  . :
+   Description . . . . . . . . . . . : 1x1 11b/g/n Wireless LAN PCI Express Half Mini Card Adapter
+   Physical Address. . . . . . . . . : E1-23-E4-CE-C5-F6
+   DHCP Enabled. . . . . . . . . . . : Yes
+   Autoconfiguration Enabled . . . . : Yes
+   IPv6 Address. . . . . . . . . . . : 2001:db8:21da:7:713e:a426:d167:37ab
+   Temporary IPv6 Address. . . . . . : 2001:db8:21da:7:5099:ba54:9881:2e54
+   Link-local IPv6 Address . . . . . : fe80::713e:a426:d167:37ab%6
+   IPv4 Address. . . . . . . . . . . : 157.60.14.11
+   Subnet Mask . . . . . . . . . . . : 255.255.255.0
+   Default Gateway . . . . . . . . . :
+   DHCPv6 IAID . . . . . . . . . . . : 232785638
+   DHCPv6 Client DUID. . . . . . . . : 00-01-00-01-1E-99-E0-56-B8-88-E3-EA-F5-7A
+   DNS Servers . . . . . . . . . . . : fec0:0:0:ffff::1%1
+                                       fec0:0:0:ffff::2%1
+                                       fec0:0:0:ffff::3%1
+   NetBIOS over Tcpip. . . . . . . . : Enabled`;
+
+describe('netsh', () => {
+    describe('_parseNetshResults', () => {
+        it('returns an object with signal values', () => {
+            const result = netsh._parseNetshResults(exampleNetshResult);
+
+            assert.equal(80, result.signal);
+            assert.equal('Wi-Fi', result.interface);
+        });
+    });
+    describe('_parseIPAddressResults', () => {
+        it('returns an object with addresses values for example 2', () => {
+            const result
+                = netsh._parseIPConfigResults(exampleIPAddressResult1, 'Wi-Fi');
+
+            assert.deepEqual([
+                    '2001:db8:21da:7:713e:a426:d167:37ab',
+                    '157.60.14.11'],
+                result);
+        });
+    });
+    describe('_parseIPAddressResults', () => {
```

Comment:
describe and it blocks can be nested. So you would only need one describe block that has two it blocks nested in it.